src/createServer.js

```diff
@@ -1,3 +1,42 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+const http = require('http');
+const { convertToCase } = require('./convertToCase');
+const { handleErrors } = require('./handleErrors');
+
+function createServer() {
+  return http.createServer((req, res) => {
+    res.setHeader('Content-Type', 'application/json');
+
+    const normalizeURL = new URL(req.url, `http://${req.headers.host}`);
+
+    const targetCase = normalizeURL.searchParams.get('toCase');
+    const originalText = normalizeURL.pathname.slice(1);
+
+    const errors = handleErrors(originalText, targetCase);
+
+    if (errors.length > 0) {
+      res.statusCode = 404;
+      res.statusMessage = 'Bad request';
+
+      res.end(JSON.stringify({
+        errors,
+      }));
+
+      return;
+    }
+
+    res.statusCode = 200;
+    res.statusMessage = 'OK';
```

Comment:
I think res.statusCode is set to 200 and the res.statusMessage is set to OK by default